pkg/kubelet/kubelet.go

```diff
@@ -1324,6 +1324,9 @@ func (kl *Kubelet) initializeModules() error {
 
 	// Start out of memory watcher.
 	if err := kl.oomWatcher.Start(kl.nodeRef); err != nil {
+		if kl.serverCertificateManager != nil {
+			kl.serverCertificateManager.Stop()
```

Comment:
Do we also want to stop `kl.imageManager`? If not, why not?

Reply:
Looking at ImageGCManager, I don't see a Stop func.
Do you know how ImageGCManager should be stopped ?

thanks

Reply:
w.r.t. 'either all modules will initialize or none will?', since Stop is missing from a few of the managers, I think the comment can be added later when the statement is true.

What do you think, @mattjmcnaughton ?